courses/free-courses-en.md

```diff
@@ -342,6 +342,7 @@
 * [Berkeley's CS  61A: Taught using SICP](https://archive.org/details/ucberkeley-webcast-PL3E89002AA9B9879E?tab=collection)
 * [Computer Networking - Network Engineering](https://www.youtube.com/watch?v=qiQR5rTSshw) - Brian Ferrill (FreeCodeCamp)
 * [CS50's Introduction to Computer Science](https://cs50.harvard.edu/x/) - David J. Malan (Harvard OpenCourseWare and edX)
+* [Discrete Mathematics for Computer Science Specialization](https://www.youtube.com/playlist?list=PL7T06JEc5PF4r-s97cCWyXFhb0PoTKg0I) - Nerd's Lesson
```

Comment:
this course is from 'my lesson' use https://www.youtube.com/playlist?list=PLtS8Ubq2bIlXO4qEM5BOsBy6xWQNVFu8l

Reply:
Changes made
Changed link from Nerd's Lesson to My Lesson